src/main/java/org/broadinstitute/consent/http/db/mapper/DraftSubmissionInterfaceMapper.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.UUID;
+import org.broadinstitute.consent.http.exceptions.NoMatchingClassException;
+import org.broadinstitute.consent.http.models.DraftInterface;
+import org.broadinstitute.consent.http.models.User;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+public class DraftSubmissionInterfaceMapper implements RowMapper<DraftInterface>, RowMapperHelper {
+
+  @Override
+  public DraftInterface map(ResultSet rs, StatementContext ctx) throws SQLException, NoMatchingClassException {
+    if (!hasColumn(rs, "schema_class")) {
+      throw new NoMatchingClassException("Missing class name.");
+    }
+
+    DraftInterface dsi;
+    String className = rs.getString("schema_class");
+
+    try {
+      dsi = (DraftInterface) Class.forName(className).getDeclaredConstructor().newInstance();
+    } catch (ClassNotFoundException | NoSuchMethodException | InvocationTargetException |
+             InstantiationException | IllegalAccessException classNotFoundException) {
+      throw new NoMatchingClassException(className);
```

Comment:
Is there a reason to not include the exception itself here? That would let the top level logger log the underlying exception and throw point.

Reply:
My thinking was generally around leaking internals to a bad actor in this particular error state.  I wanted to communicate the condition but not the details.  If there's no need to worry about that for this, then I think the way to resolve this would be to bubble up the exception.